### ISSUE TITLE
Sprout reports TC Error pjsip: Assert failed: ../src/pjsip/sip_transport.c:1022 tp->idle_timer.id == 0

### ISSUE THREAD

**Issue by tcaterisano** [Metaswitch/sprout#1385]
#### Symptoms

Sprout reports TC Error pjsip: Assert failed: ../src/pjsip/sip_transport.c:1022 tp->idle_timer.id == 0

#### Impact
Sprout overloading at low call rates

#### Release and environment 
sprout                         9.0.0-1-1.0.0-160329.193924

#### Steps to reproduce

traffic at 10 cps

**Comment by johadalin** [comment]
Adding a little more detail:

The log text reads:
04-04-2016 20:01:32.029 UTC Error pjsip: Assert failed: ../src/pjsip/sip_transport.c:1022 tp->idle_timer.id == 0
04-04-2016 20:01:32.029 UTC Error pjsip:        timer.c Timer was scheduled more than once: internal ID 19, user-provided ID 1, delay 33 sec, 0 msec, user data 0x7f77006f0f48, callback 0x66b6f2, group lock (nil)
04-04-2016 20:01:32.029 UTC Error pjsip: Assert failed: ../src/pj/timer.c:504 entry->_timer_id < 1

in each case the 'internal ID' is the only part that changes.

They don't appear to happen at any regular interval, and currently do not appear to be causing call issues.
@AlexHockey Did you see anything like this in your analysis of pjsip, or your work on pjsip timers (as per https://github.com/Metaswitch/sprout/issues/168#issuecomment-194795798)

**Comment by AlexHockey** [comment]
I've not seen this before I'm afraid. I don't think #168 is relevant, as that issue was about a timer popping at the same time as it was cancelled.

**Comment by mirw** [comment]
@tcaterisano, please can you confirm whether "sprout overloading at low call rates" really is caused by this?  If so, I think this is critical.  If not, I'd argue not.

**Comment by johadalin** [comment]
I've discussed with Terry, and we agree that this is likely not the cause of the overloading error reports. As such, this is not critical unless we see otherwise. 
We will keep a lookout for these errors and try to correlate them to a likely cause.

**Comment by tcaterisano** [comment]
No longer seeing this error. Corrective action included:

1)	Perimeta 3.8_40                         - eliminated (3.9 503 errors)
2)	max_tokens="100" (default 20)   -eliminated (sprout/homestead overload) 503
3)	hussy Cx + OpemIMS HSS sh/zh/xcap  - eliminated (480 – hss timeouts with openIMS Hss)
4)     raised DNS TTL to 600

**Comment by mirw** [comment]
I don't think we should close this - it's still a bug.  I think we've concluded that it shouldn't be critical, though.
